lib/ansible/executor/stats.py

```diff
@@ -73,6 +74,7 @@ def summarize(self, host):
 
     def set_custom_stats(self, which, what, host=None):
         ''' allow setting of a custom stat'''
+        validate_variable_names(which, 'stats')
```

Comment:
not sure stats need to be validated with the same criteria, as they are not really normal variables, but things we pass into tower for workflows.